quiz/backend/helpers/gen_username/generate.py

```diff
@@ -0,0 +1,20 @@
+import random
+
+
+def generate_username(num_results=5):
+    adjectives, nouns = [], []
+    with open("data/useradj.txt", "r") as file_useradj:
+        for line in file_useradj:
+            adjectives.append(line.strip())
+    with open("data/usernoun.txt", "r") as file_usernoun:
+        for line in file_usernoun:
+            nouns.append(line.strip())
+
+    usernames = []
+    for _ in range(num_results):
+        adjective = random.choice(adjectives)
+        noun = random.choice(nouns)
+        num = str(random.randrange(100))
+        usernames.append(adjective + noun + num)
+
+    return usernames
```

Comment:
I've got two thoughts
1. List/Set comprehension for lines 14-20 if you want to stick to yielding a list

2. turning entire generate_username into generator
```
while True:
    adjective = random.choice(adjectives)
    noun = random.choice(nouns)
    num = str(random.randrange(100))
    yield f"{adjective}{noun}{num}"
```

3. Considering theres 330 and 340 words in each set, it gives us 112_000 unique nicknames adding numbers may be an overkill

Reply:
for me, making generator for sending grouped data to the frontend, like 5 items at once is unnecessary